app/(auth)/layout.tsx

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable tailwindcss/no-custom-classname */
+import React from "react";
+import Image from "next/image";
+const AuthLayout = ({ children }: { children: React.ReactNode }) => {
+  return (
+    <main className="min-w-screen min-h-screen bg-black-900">
```

Comment:
```suggestion
<main className="auth-background">
```
The login auth pages will be simple enough. They won't need to scroll so setting h-full here is fine.

Reply:
Ok I'll go that route